deployment_utils.py

```diff
@@ -107,3 +108,14 @@ def tribler_is_installed():
         return False
 
     return True
+
+
+def init_sentry():
+    """ Inits the sentry sdk by setting up a URL and a trace sample rate
+
+    To change the URL, please specify a corresponding ENV variable.
+    """
+    sentry_sdk.init(
```

Comment:
Two points:
1. Maybe leaving out default URL is better. Since this repo is public and anybody can run these scripts, we could get false reports or spams. We could set the correct URL in jenkins job parameter for our use case.
2. Catching exceptions if sentry cannot initialize might be better so anybody without a working sentry URL can run without issues.

Reply:
Thank you for this suggestions. Will implement them.

Reply:
Done